MSCL/source/mscl/MicroStrain/MIP/MipNode_Impl.cpp

```diff
@@ -2191,6 +2191,29 @@ namespace mscl
         return r.parseResponse(doCommand(r, ActivationCode::buildCommand_get()));
     }
 
+    EventSupportInfo MipNode_Impl::getEventInfo(const EventSupportInfo::Query query) const
+    {
+        const MipFieldValues response = get(MipTypes::Command::CMD_EVENT_SUPPORT,
+                    { Value::UINT8(static_cast<uint8>(query)) }
+        );
+
+        EventSupportInfo info{};
+
+        info.query = static_cast<EventSupportInfo::Query>(response[0].as_int8());
+        info.maxInstances = response[1].as_int8();
+
+        const uint8 count = response[2].as_int8();
```

Comment:
I would rename this variable to `num_entries` to match with the MIP definition

Reply:
I think we're sticking to lower camel case in MSCL, but I did rename it to match the MIP doc